todo-dnd/todo/stories/TodoListApp/TodoList.stories.js

```diff
@@ -0,0 +1,45 @@
+import { TodoInputView } from '../../src/todo/TodoInputView';
+import { TodoListView } from '../../src/todo/TodoListView';
+import { TodoStatusView } from '../../src/todo/TodoStatusView';
+import '../../src/css/todo.css';
+
+export default {
+  title: 'Todo List App',
+};
+
+export const NoTodoItem = () => {
+  const todoInputView = new TodoInputView();
```

Comment:
`TodoListApp`를 사용하시는게 편하실텐데 따로 view를 각각 가져와서 사용하신 이유가 있나요?

아래처럼 하면 storybook의 args 기능도 사용할 수 있습니다.
```suggestion
const Template = ({ useDnd }) => {
  const rootEl = document.createElement('div');
  const todoListApp = new TodoListApp(rootEl, useDnd);
  todoListApp.render();

  return rootEl;
};

export const Default = Template.bind({});
Default.args = {
  useDnd: false,
};
```

Reply:
9b89201e515ee3266558951cfa5822da83e094f0 스토리 작성 시 TodoListApp을 사용할 생각을 못했었던 것 같아요. 제안주신 내용 참고하여 수정하였습니다!